src/math/Vector2.js

```diff
@@ -380,9 +380,7 @@ Object.assign( Vector2.prototype, {
 
 		// computes the angle in radians with respect to the positive x-axis
 
-		var angle = Math.atan2( this.y, this.x );
-
-		if ( angle < 0 ) angle += 2 * Math.PI;
+		var angle = Math.atan2( - this.y, - this.x ) + Math.PI;
```

Comment:
I think it would be good to implement the unit test for this method:

https://github.com/mrdoob/three.js/blob/d69f9a7d7cac55a4674639f259275a5119f1bc17/test/unit/src/math/Vector2.tests.js#L363-L367

In this way, it's easier to verify if the current and new implementation actually produce the same result.

Reply:
I'll try how to use QUnit.